### DISCUSSION TITLE
Setting RPI Tuning File

### DISCUSSION THREAD

**Discussion by shadmanhabib** [bluenviron/mediamtx#1256] [Questions]
Hey guys, had a quick question regarding how tuning files for Raspberry Pi cameras need to be setup. My tuning file is located at `/usr/share/libcamera/ipa/raspberrypi/imx219_120.json`.

I know it's the `rpiCameraTuningFile` value that I need to set. But I'm not certain as to the format that I need to enter in.

Is it `rpiCameraTuningFile: /usr/share/libcamera/ipa/raspberrypi/imx219_120.json` for example? Or do I need to add quotations marks? Do I just need to put in the file name as in `rpiCameraTuningFile: imx219_120`.

I've tried a few ways including the examples above of entering the file in but haven't been able to get it to work. Any help would be great.

**Comment by gofaster** [comment]
Hello,  have the same question.  No matter a valid or non-existent path; the image characteristics do not change and no error is generated.

`rpiCameraTuningFile: '/usr/share/libcamera/ipa/raspberrypi/imx219.json'`
`rpiCameraTuningFile: '/foo.bar'`

I would appreciate some clue.

**Comment by aler9** [comment]
`rpiCameraTuningFile` was not applied correctly; this is fixed with #1345

[a comment by an automated account is not shown]
